starter-kits.json

```diff
@@ -14,5 +14,6 @@
   "solidjs-tailwind": "SolidJS and TailwindCSS",
   "solidstart-tanstackquery-tailwind-modules": "Solid Start, TanStack Query and Tailwind CSS with CSS Modules",
   "svelte-kit-scss": "SvelteKit and SCSS",
-  "vue3-apollo-quasar": "Vue3, Apollo, and Quasar"
+  "vue3-apollo-quasar": "Vue3, Apollo, and Quasar",
+  "next13-zustand-bulma": "NextJS 13, Zustand, and Bulma"
```

Comment:
I think we need to run the `yarn generate-starter-kits-json`.
I think it supposed to sort the starters in order.